lib/pages/task_list.dart

```diff
@@ -17,90 +19,68 @@ class _TaskListState extends State<TaskList> {
     context.read<TaskCubit>().readTasks();
     return BlocBuilder<TaskCubit, TaskState>(
       builder: (context, state) {
-        List<Widget> tasks = state.taskNames.map(
-          (taskName) {
+        List<Widget> tasks = state.tasks.map(
+          (task) {
             return ListTile(
-              leading: Icon(
-                Icons.task,
-                color: iconColor,
+              key: const Key('task tiles'),
```

Comment:
I think in general you don't want `Key`s with spaces. I would try sticking to a consistent "_" or "-" to take the place of spaces.